content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -260,6 +260,38 @@ problems, the kubelet resets the restart backoff timer for that container.
 [Sidecar containers and Pod lifecycle](/docs/concepts/workloads/pods/sidecar-containers/#sidecar-containers-and-pod-lifecycle)
 explains the behaviour of `init containers` when specify `restartpolicy` field on it.
 
+### Configurable container restart delay
+
+{{< feature-state feature_state_name="KubeletCrashLoopBackOffMax" >}}
```

Comment:
it seems that alpha is not rendering properly

- preview: https://deploy-preview-48499--kubernetes-io-main-staging.netlify.app/docs/concepts/workloads/pods/pod-lifecycle/#configurable-container-restart-delay

Reply:
Ah I think it had the wrong key (`state` instead of `gate`). Looks good now at https://deploy-preview-48499--kubernetes-io-main-staging.netlify.app/docs/concepts/workloads/pods/pod-lifecycle/#configurable-container-restart-delay after the fix in 333e896